sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/DefaultFileSystem.java

```diff
@@ -172,15 +160,13 @@ public boolean hasFiles(FilePredicate predicate) {
 
   @Override
   public Iterable<File> files(FilePredicate predicate) {
-    doPreloadFiles();
     return () -> StreamSupport.stream(inputFiles(predicate).spliterator(), false)
       .map(InputFile::file)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "file"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)